pkg/sypgp/sypgp.go

```diff
@@ -461,6 +483,28 @@ func removeKey(list openpgp.EntityList, fingerprint string) openpgp.EntityList {
 	return nil
 }
 
+// RemovePrivKey will delete a secret key matching toDelete
+func (keyring *Handle) RemovePrivKey(toDelete string) error {
+	// read all the local private keys
```

Comment:
We need to error out with an appropriate message if there is an attempt to remove a private key from the global keyring, as the global keyring only holds public keys (see the appendPrivateKey)